src/classes/QueryArgumentFormatter.cls

```diff
@@ -6,42 +6,43 @@ public without sharing class QueryArgumentFormatter {
 
     private Object argumentValue;
 
-    public QueryArgumentFormatter(Object argumentValue) {
+    private QueryArgumentFormatter(Object argumentValue) {
         this.argumentValue = argumentValue;
     }
 
-    public String getValue() {
+    private String getValue() {
         return this.objectToQueryString(this.argumentValue);
     }
 
+    public static String VALUE(Object argumentValue) {
```

Comment:
I'd just scrap this method and make getValue a static method (and eliminate the constructor)